packages/sampler/stories/qa/VirtualGridList.js

```diff
@@ -58,17 +64,20 @@ storiesOf('VirtualGridList', module)
 			<VirtualGridList
 				dataSize={updateDataSize(number('dataSize', Config, defaultDataSize))}
 				direction="horizontal"
-				focusableScrollbar={boolean('focusableScrollbar', Config, false)}
-				horizontalScrollbar={select('horizontalScrollbar', ['auto', 'hidden', 'visible'], Config, 'auto')}
+				focusableScrollbar={boolean('focusableScrollbar', Config)}
+				horizontalScrollbar={select('horizontalScrollbar', ['auto', 'hidden', 'visible'], Config)}
```

Comment:
Please define prop.horizontalScrollbar of `['auto', 'hidden', 'visible']` and use it.

Reply:
Your idea is good for code consistency but my concern is that the array is used only one time at that point.

Reply:
Got it.